### DISCUSSION TITLE
Are the "python-colormath" observers and illuminants combinations highlighting actual industry usage?

### DISCUSSION THREAD

**Discussion by blue-j** [colour-science/colour#1108] [General]
I am trying to verify the claims made on this page:

https://python-colormath.readthedocs.io/en/latest/illuminants.html

Are these observer/illuminant combinations intended generally, or merely reflective of the software's capacities?

Thank you in advance!  : )

- J

**Comment by thomasmansencal** [comment]
Hi @blue-j,

The latter, illuminants are typically defined mathematically or with tabulated spectral data so their tristimulus values can be computed for any observer. Which observer and angle you use depends on which industry you are working into, we tend to use the CIE 1931 2 Degree Standard Observer in the VFX industry.,

Cheers,

Thomas

**Reply by blue-j** [reply]
Thank you, sensei!  : )
